hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -776,4 +790,90 @@ public static ContainerProtos.ContainerCommandResponseProto processForDebug(
 
     return msg;
   }
+
+  /**
+   * Use raft client to send admin request, transfer the leadership.
+   * If not isRandom, then use host to match the target leader.
+   * 1. Set priority and send setConfiguration request
+   * 2. Trigger transferLeadership API.
+   *
+   * @param server        the Raft server
+   * @param groupId       the Raft group Id
+   * @param host          the string of the host, should be IP:PORT format
+   * @param isRandom      whether to choose random follower as target leader
+   * @param curLeader     the current Raft leader
+   * @throws IOException
+   */
+  public static void transferRatisLeadership(RaftServer server,
```

Comment:
I think extracting these methods in LeadershipManager class would be more useful than keeping them in a util class.

Reply:
Could you tell me why it would be more useful? The process is short-term and hence the LeadershipManager will be temporary. The cost of creating a such class is a bit wasting I think.

Reply:
I'm out of context a bit. Why is the process short-term?

Reply:
We mainly use the Raft client to trigger two admin APIs,  setConfiguration and transferLeadership. Such the process is short-term and straightforward, so we need not keep any state which needs to hold the class.